store/queries/post_hotness.sql

```diff
@@ -0,0 +1,24 @@
+-- name: DeleteOldPostHotness :execrows
+DELETE FROM post_hotness
+WHERE generated_at < NOW() - sqlc.arg(retention_period)::INTERVAL;
```

Comment:
Rather than expressing retention as a time period, maybe it makes more sense to express it as "Keep the last X"? This means we won't need to adjust the retention period if we adjust the generation period.